cmd/compose/docker-compose.go

```diff
@@ -12,6 +12,11 @@ import (
 // DockerComposeImage holds the Docker image:tag to use for Docker Compose
 const DockerComposeImage = "docker/compose:1.28.0"
 
+// TtyAware interface holds functions for becoming aware of TTY
+type TtyAware interface {
```

Comment:
why do we need a separated interface for it?

Reply:
so we can have a generic check - the conditional for knowing when to inject it... before it was done using the struct pointer itself. This way, with the interface, we can mock/replace during tests.

I started but aborted writing such tests, but kept this part as it is a step closer on that direction.